src/screens/forgotPassword/ForgotPassword.jsx

```diff
@@ -60,21 +60,25 @@ const ForgotPassword = () => {
   useEffect(() => {
```

Comment:
this block is still not showing latest message. It shows one message but it might still be old message that is showing

Reply:
Fixed